### PR TITLE
Remove multisig

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: runtime/crab/src/lib.rs [hunk 1]
```diff
@@ -144,7 +144,6 @@ frame_support::construct_runtime! {
 		Scheduler: pallet_scheduler = 28,
 		Preimage: pallet_preimage = 29,
 		Proxy: pallet_proxy = 30,
-		Multisig: pallet_multisig = 31,
```
Comment (boundless-forest) [inline runtime/crab/src/lib.rs:147]: Do we need to update the pallet prefix after 31?

Comment (AurevoirXavier) [reply runtime/crab/src/lib.rs:147]: I suggest updating this in the finaln release.

Comment (boundless-forest) [reply runtime/crab/src/lib.rs:147]: Please do it right now or create an issue.

Comment (AurevoirXavier) [reply runtime/crab/src/lib.rs:147]: There is already an issue tracking this.

**Review by boundless-forest — APPROVED** [review]

**Pull request merged**